### PR TITLE
fix: golangci-lint error

### PR DESCRIPTION
**Pull request by FrankYang0529**
#### Which issue(s) this PR fixes:
https://github.com/longhorn/longhorn/issues/7448

### CODE REVIEW

File: pkg/datasource/service.go [hunk 1]
```diff
@@ -158,36 +158,33 @@ func (s *Service) waitForBeginning() {
...
-				return
+		<-ticker.C
+		if count >= RetryCount {
+			s.lock.Lock()
+			s.timeoutBegin = true
+			s.dsInfo.State = string(types.StateFailed)
+			s.dsInfo.Message = TimeoutBeginErrorMessage
+			s.lock.Unlock()
+			if err := s.syncClient.Delete(s.filePath); err != nil {
+				s.log.Errorf("DataSource Service: failed to do cleanup after timeout waiting for the datasource processing begin: %v", err)
```
Comment (derekbit) [inline pkg/datasource/service.go:169]:
```suggestion
				s.log.Errorf("DataSource Service: failed to do cleanup after timeout waiting for the datasource processing begin: %v", err)
```
```suggestion
				s.log.WithError(err).Error("DataSource Service: failed to do cleanup after timeout waiting for the datasource processing begin")
```

File: pkg/manager/service.go [hunk 2]
```diff
@@ -147,7 +147,9 @@ func (m *Manager) monitoring() {
 			done = true
 			break
 		case <-ticker.C:
-			m.listAndUpdate()
+			if _, err := m.listAndUpdate(); err != nil {
+				m.log.WithError(err).Error("failed to list and update backing image files")
```
Comment (derekbit) [inline pkg/manager/service.go:151]:
```suggestion
				m.log.WithError(err).Warn("failed to list and update backing image files")
```

**Comment by FrankYang0529** [comment]
All comments are addressed. Thank you.

**Review by derekbit — APPROVED** [review]

**Comment by FrankYang0529** [comment]
@mergify backport v1.6.x

**Comment by FrankYang0529** [comment]
@mergify backport v1.5.x

**Pull request merged**